Enterprise-CMCS/macpro-mdct-mfp
# start bringing over PDF components from MCR

### Description

The second step in creating the WP PDF - porting over PDF components from MCR. 

There is probably a component or two that I won't end up needing, but I'm bringing them over now, and will delete later if I don't need them, or can't repurpose them.

### Related ticket(s)


CMDCT-218

---
### How to test

You can create a workplan and then click Review PDF to be taken to the PDF export - there are a few things showing up, and hopefully no breaking errors.
You can fill out a WP and click Review PDF to see the export, which should have a little bit more info and components to see, and also no breaking errors.

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
